alembic/versions/570f4ea99cda_add_labels_to_proposal_space.py

```diff
@@ -0,0 +1,30 @@
+import json
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import table, column
+from coaster.sqlalchemy import JsonDict
+from funnel.models.space import labels_default
```

Comment:
Migration must never reference code. This migration may be run on a future version of the Funnel codebase where `labels_default` is no longer defined. It should still work.

Also, `server_default` is not recommended (see comment below), so you can just remove this.